### ISSUE TITLE
Compatibility Notification center 1.7

### ISSUE THREAD

**Issue by Kahmoon** [codefog/contao-events_subscriptions#96]
Hi Kamil,

your module seems to be incompatible of missmatching dependencies with the latest version of notification center.

`
Problem 1
    - codefog/contao-haste[4.16.0, ..., 4.20.3] require php ~5.4 || ~7.0 -> your php version (8.1.13.0) does not satisfy that requirement.
    - codefog/contao-haste[4.21.0, ..., 4.24.13] require php ~5.6 || ~7.0 -> your php version (8.1.13.0) does not satisfy that requirement.
    - Root composer.json requires frank-berger/events-subscriptions 1.2.2 -> satisfiable by frank-berger/events-subscriptions[1.2.2].
    - codefog/contao-events_subscriptions[2.15.0, ..., 2.16.3] require codefog/contao-haste ~4.16 -> satisfiable by codefog/contao-haste[4.16.0, ..., 4.25.23].
    - You can only install one version of a package, so only one of these can be installed: codefog/contao-haste[4.5.1, ..., 4.25.23, 5.0.0, ..., 5.0.5].
    - terminal42/notification_center 1.7.0 requires codefog/contao-haste ^5.0 -> satisfiable by codefog/contao-haste[5.0.0, ..., 5.0.5].
    - frank-berger/events-subscriptions 1.2.2 requires codefog/contao-events_subscriptions ^2.15 -> satisfiable by codefog/contao-events_subscriptions[2.15.0, ..., 2.16.3].
    - Root composer.json requires terminal42/notification_center ^1.7 -> satisfiable by terminal42/notification_center[1.7.0].
`

Greetz

**Comment by fritzmg** [comment]
`codefog/contao-events_subscriptions` is not compatible with `codefog/contao-haste: 5.x`. You need to change your `terminal42/notification_center` requirement to `^1.6`.

**Comment by Kahmoon** [comment]
Its actually on 1.6. I wanted to update :)

**Comment by fritzmg** [comment]
> Its actually on 1.6. I wanted to upgrade :)

Yes, you can't at this time.

**Comment by Kahmoon** [comment]
nope...i just want my installation to be as up to date as possible.

**Comment by fritzmg** [comment]
Composer always provides the most up to date versions that are possible for your requirements. You do not need to manually change your requirements - unless you want to switch to a new major version of a package.

**Comment by Kahmoon** [comment]
Yep, in this case this would be a (wanted) major version change.

**Comment by fritzmg** [comment]
But there is no new major version for the Notification Center.

**Comment by Kahmoon** [comment]
I meant version 1.7. When this module gets ready for haste 5, this case will be closed so i can update afterwards.

First of all i thought its a problem with our custom sub-module

**Comment by fritzmg** [comment]
> I meant version 1.7.

That's a minor version, not a major version. Also I thought you did not need the new feature version?
